.drone.star

```diff
@@ -1550,7 +1550,7 @@ def collaboraService():
             "detach": True,
             "environment": {
                 "DONT_GEN_SSL_CERT": "set",
```

Comment:
is it ok, that the value is different, that the value in the docker compose ?

Reply:
I don't think it's related, but we could try changing it to “yes”.

Reply:
if we use "extra_params":`--o:ssl.enable=false` -> we need to switch using `http` so https://github.com/owncloud/web/pull/11301/files#diff-6a968d4b7901c8526c8d9a86d1f434737dbfe2b00da21a7d3041ed4e3d58d62eL1598-L1599
should be: 
```
environment["COLLABORATION_APP_ADDR"] = "http://collabora:9980"
        environment["COLLABORATION_APP_ICON"] = "http://collabora:9980/favicon.ico"
```
right?

Reply:
I don't think we need all those changes, just bump code version in drone and hope for the best 😮

Reply:
I'll try. if it doesn't work, we'll do it in a separate PR

Reply:
<img width="558" alt="Screenshot 2024-07-30 at 15 53 14" src="https://github.com/user-attachments/assets/da9e5149-bb91-47b3-9193-10495f51db7a">

tests are green due the new popup. I'll look how disable popup in the separate PR

Reply:
Whooopsie. Thanks for the engagement 👍